integrations/datadog/README.md

```diff
@@ -26,7 +26,7 @@ This demo shows how to make Materialize metadata available as key metrics for mo
     cd integrations/datadog
     ```
 
-2. Edit the `config.yaml` file and set your Materialize details under the `connections` key:
+2. Edit the `config.yaml` file and set your Materialize details under the two available `connections` keys:
```

Comment:
Would it be possible to use environment variables instead of asking users to manually replace the values? Now that this needs to be done in two different places, it'd be a less complicated approach.